Makefile

```diff
@@ -20,8 +20,7 @@ VDE_INSTALL ?= /opt/finch
 UNAME := $(shell uname -m)
 ARCH ?= $(UNAME)
 SUPPORTED_ARCH = false
-CORE_VERSION ?= 0.1.1
-CORE_URL ?= https://artifact.runfinch.com/finch-core-$(CORE_VERSION).tar.gz
+CORE_URL ?= https://artifact.runfinch.com/finch-core-0.1.1.tar.gz
```

Comment:
We can still keep this env for easy core version bump for local development and test?

Reply:
https://github.com/runfinch/finch/blob/ba12834325006e60f37939a0f819bfe3686b5461/.github/bin/update-lima-dep.sh#L38 fetches the latest `finch-core` from s3 which would be of the format finch-core-<version>.tar.gz . It's possible to extract out the version from here but it may be cumbersome in case the format of the filename changes or renamed in s3. If the user wants to change the version it's still changing only in one place although not as convenient as having an env for just the version.  wdyt?

Reply:
Users can still pass their own `CORE_URL` variable, instead of just `CORE_VERSION`. This is the only place `CORE_VERSION` was used anyway, so the only difference is now users have to specify the entire URL instead of just the version. That's how I usually used CORE_URL anyway, since it works with `file://` URLs (its usually easier to use a local file URL in-dev than an internet/S3 URL anyway).

Reply:
Yeah so this way users can override the `CORE_URL` by `make <target> CORE_URL=<custom url>`. So we don't really need the `CORE_VERSION` variable.